packages/firebase_auth/firebase_auth/test_driver/firebase_auth_e2e_test.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020, the Chromium project authors.  Please see the AUTHORS file
```

Comment:
I think this file needs to be moved to the example app. Does it work if you do `flutter drive test/firebase_auth_e2e.dart` now?

Reply:
I can't due to the gradle issue we've talked about. But if I moved it to example, it complains no test file found.